src/lib/collectors/shared/launcher.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @fileoverview Launches the given browser with the right configuration to be used via the Chrome Debugging Protocol
+ *
+ * Supported browsers: Chrome
+ *
+ * This is a mix between:
```

Comment:
I think you can remove this notice since the code here is no longer coming from there and the code from lighthouse is now in the Chrome launcher dependency.